pages/content_packs.rst

```diff
@@ -66,6 +66,16 @@ How do I create a Content Pack?
    take a close look if all needed configurations are included.
```

Comment:
Tempted to suggest you change "needed" to "needful" ;-)

Reply:
That would have been the needful, I guess :smile: